concepts/index.md

```diff
@@ -101,14 +101,18 @@ pipeline start/complete/fail).
 
 ### Stage
 
-A *Stage* in Spinnaker is an atomic building block for a pipeline, describing an
-action that the pipeline will perform. You can sequence
-[stages](/reference/pipeline/stages/) in a Pipeline in any order, though some
+A *Stage* in Spinnaker is a collection of sequential Tasks and composed Stages that
```

Comment:
What do we mean by "composed stages" making up a stage?

Reply:
ditto